win32/build/mkdist.php

```diff
@@ -271,7 +271,7 @@ function extract_file_from_tarball($pkg, $filename, $dest_dir) /* {{{ */
 }
 
 /* include a snapshot identifier */
-$branch = "HEAD"; // TODO - determine this from SVN branche name
+$branch = "HEAD"; // TODO - determine this from SVN branch name
```

Comment:
Might as well fix the mention of SVN at the same time

Reply:
```suggestion
$branch = "HEAD"; // TODO - determine this from GitHub branch name
```